test/integration/outbound/pisp-otp-validate.test.ts

```diff
@@ -0,0 +1,100 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the "License")
+ and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed
+ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Kevin Leyow - kevin.leyow@modusbox.com
+ --------------
+ ******/
+import axios from 'axios'
+import env from '../env'
+import { KVS } from '~/shared/kvs'
+import { RedisConnectionConfig } from '~/shared/redis-connection'
+import Config from '~/shared/config'
+import mockLogger from '../../unit/mockLogger'
+import { PISPOTPValidateModelState } from '~/models/outbound/pispOTPValidate.interface';
+
+describe.only('PISP OTP Validate', (): void => {
+  const config: RedisConnectionConfig = {
+    host: Config.REDIS.HOST,
+    port: Config.REDIS.PORT,
+    logger: mockLogger(),
+    timeout: Config.REDIS.TIMEOUT
+  }
+  let kvs: KVS
+  const consentRequestsRequestId = 'ab049ac1-b936-4327-ab76-3e66e608beb8'
```

Comment:
I would add here a comment that these values are used by TTK to simulate proper responses

Reply:
Wouldn't it be better to use `authToken` value to switch between simulation cases in TTK instead of using `consentRequestId`? 

Then we can use `uuid()` function to generate id.

Unique id for every call is a part of the interface protocol,
and we didn't test this requirement when using hardcoded ones.

Using unique id will improve our test coverage

Reply:
done and switched to using authToken as the rule.